src/pre_process/m_data_output.fpp

```diff
@@ -767,15 +785,40 @@ contains
 
         end if
 
-        open (1, FILE='equations.dat', STATUS='unknown')
+        open (1, FILE='indices.dat', STATUS='unknown')
+
+        write (1, '(A3,A20,A20)') "#", "Conservative", "Primitive"
+        write (1, '(A)') "-------------------------------------------"
+        do i = contxb, contxe
+            write (temp, '(I0)') i - contxb + 1
+            write (1, '(I3,A20,A20)') i, "\alpha_{"//trim(temp)//"} \rho_{"//trim(temp)//"}", "\alpha_{"//trim(temp)//"} \rho"
+        end do
+        do i = momxb, momxe
+            write (1, '(I3,A20,A20)') i, "\rho u_"//coord(i - momxb + 1), "u_"//coord(i - momxb + 1)
+        end do
+        do i = E_idx, E_idx
+            write (1, '(I3,A20,A20)') i, "\rho U", "p"
+        end do
+        do i = advxb, advxe
+            write (temp, '(I0)') i - contxb + 1
+            write (1, '(I3,A20,A20)') i, "\alpha_{"//trim(temp)//"}", "\alpha_{"//trim(temp)//"}"
+        end do
+        if (chemistry) then
```

Comment:
if there are physics involved or some other features then there could be other equations. subgrid bubbles is one such example.